src/app/pages/courses-page/actions-section/actions-section.component.html

```diff
@@ -1 +1,14 @@
-<p>actions-section works!</p>
+<div class="actions-section-container">
+  <div class="search-course-container">
+    <div class="form-group">
+      <input type="text" placeholder="Type to search" class="form-control" [(ngModel)]="searchString">
+    </div>
+    <button class="button button--large button--green" (click)="searchCourseHandler()">Search</button>
+  </div>
```

Comment:
this can be a separate independent shared component. `SearchInput` or something lile that

Reply:
Agree, makes sense